### DISCUSSION TITLE
checkboxes in Dark theme not really visible

### DISCUSSION THREAD

**Discussion by atsushieno** [community/community#4384] [Accessibility] [answered]
I have been using issues to track development effort, with checkboxes to track progress. And I use Dark theme to reflect desktop settings. Sadly those two don't seem to blend. This is how my issues entry looks like:

![image](https://user-images.githubusercontent.com/53929/123735292-eff13f80-d8d9-11eb-98e8-0be0c36cddea.png)

(a1), (b1a) and (b1b) are checked, but they don't look different from unchecked boxes. Maybe there can be better distinction I guess?

(update) I forgot to add basic info: It is on Firefox 89.0.2 on Ubuntu 20.04 (using `Materia-dark` theme on desktop)

**Comment by cehfisher** [comment]
Thank you @atsushieno for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again!

**Comment by atsushieno** [comment, marked as answer]
obsolete issue.
